CHANGELOG.md

```diff
@@ -489,7 +489,19 @@ Non-backwards compatible changes
   * `fold-*`
   * `fold-pull`
 
-* In `Data.Fin.Properties` the `i` argument to `opposite-suc` has been made explicit.
+* In `Data.Fin.Properties`:
+  + the `i` argument to `opposite-suc` has been made explicit;
+  + `pigeonhole` has been strenghtened: wlog, we return a proof that `i < j` rather
+    than a mere `i ≢ j`.
+
+* In `Data.Fin.Induction`:
```

Comment:
This particular entry isn't a breaking change, so should go under the `Other minor changes` section of the CHANGELOG instead of the breaking changes sections. Other than that, it all looks good, so will merge it in after this :+1:

Reply:
Fixed. (I assume you mean the entry about `Data.Fin.Induction`, not the one about `pigeonhole`, which is breaking AFAIK).

Reply:
Yup, thanks!